### ISSUE TITLE
Accessibility Checklist Missing

### ISSUE THREAD

**Issue by sstevens2** [carpentries/instructor-training#885]
The link for the [accessibility checklist](https://software-carpentry.org/checklists/accessibility/) in the [motiviation and demotiviation](https://carpentries.github.io/instructor-training/08-motivation/index.html) section leads to a blank page.  I'm guessing this page existed at one point, maybe this is due to an update in the handbook?

**Comment by maneesha** [comment]
Thanks @sstevens2 !  This should link to https://docs.carpentries.org/topic_folders/hosts_instructors/workshop_needs.html#accessibility

Fixed with 5dc7845ec2c5802bba4b450acd972fdf8fed664b
